lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
+
+  if(currentOS === 'mac') {
+    this.app.state.uploads = _.reject(this.app.state.uploads, function(d){
+      return d.key === 'precisionxtra'; // TODO: || d.key === 'abbottfreestylelite' || d.key === 'abbottfreestylefreedomlite'
```

Comment:
Do you think as we add more and more devices we should actually just add an attribute to each device's object in `this.app.state.uploads` specifying which OSes it's *not* compatible with and use that for the filtering?